start_coupled.R

```diff
@@ -68,9 +27,6 @@ start_coupled <- function(path_remind, path_magpie, cfg_rem, cfg_mag, runname, m
     return(out)
   }
 
-  # start coupling in debug mode (just create empty results folders and copy dummy reports without running the models)
-  debug <- FALSE
```

Comment:
So the mode was simply activated by setting `debug = TRUE` here?

Reply:
Yes. But the new opportunity witht the empty models is much more useful.